src/openvslam_ros.h

```diff
@@ -44,11 +45,19 @@ class stereo : public system {
 public:
     stereo(const std::shared_ptr<openvslam::config>& cfg, const std::string& vocab_file_path, const std::string& mask_img_path,
            const bool rectify);
-    void callback(const sensor_msgs::msg::Image::ConstPtr& left, const sensor_msgs::msg::Image::ConstPtr& right);
+    void callback(const sensor_msgs::msg::Image::ConstSharedPtr& left, const sensor_msgs::msg::Image::ConstSharedPtr& right);
```

Comment:
Did you perform a run-time verification of this (I mean Gazebo TB3 simulation, synthetic or ROSBAG topics publishing or something else)?